gluecodium/src/main/resources/templates/jni/utils/JniClassCacheHeader.mustache

```diff
@@ -41,7 +44,7 @@ public:
     static void init(JNIEnv* env);
 
 protected:
-    static jni::JniReference<jclass>* get_java_class_for_instance(const std::string& id);
+    static jni::JniReference<jclass>* get_java_class_for_instance(const TypeId id);
```

Comment:
Fields of this class (m_name, m_cpp_name) and constructor parameters may also be std::string_view now (in separate commit?)

Reply:
There is no benefit to it. These fields are initialized with static string literals. I would even expect using `string_view` would take more memory than just a `const char*`.